docs/providers/aws/guide/serverless.yml.md

```diff
@@ -83,6 +83,8 @@ provider:
   # Optional CloudFormation tags to apply to the stack
   stackTags:
     key: value
+  # Method used during `deploy` for CloudFormation stack deployments (default: changesets), can be 'changesets' or 'direct'
```

Comment:
```suggestion
  # Method used for CloudFormation deployments: 'changesets' or 'direct' (default: changesets)
  # See https://www.serverless.com/framework/docs/providers/aws/guide/deploying#deployment-method
```

to fit on the screen, and also to match other sections that also link to the docs with more details.

Reply:
thanks 🙇